cardano-rt-view/test/runTest.sh

```diff
@@ -17,31 +25,35 @@ echo "Remove old UNIX-socket..."
 rm -f "${UNIX_SOCKET}"
 
 echo "Launch cardano-rt-view-service..."
-nohup stack exec -- "${RT_VIEW_EXE}" --config "${RT_VIEW_CONFIG}" \
-                                     --static "${RT_VIEW_STATIC_DIR}" \
-                                     --port "${RT_VIEW_WEB_PORT}" &
```

Comment:
Why did you remove `nohup`? It's truly daemonizes the `cardano-rt-view-service` process, I thought it's better solution.

Reply:
can't use it with the "run" script function.